### ISSUE TITLE
Bring over to Bookdown proper?

### ISSUE THREAD

**Issue by pmjones** [tobiju/bookdown-bootswatch-templates#32]
How would you feel about making this an official Bookdown project under the same namespace, github, etc?  You'd stay fully in charge of it, of course.

**Comment by tobiju** [comment]
Sounds good. Then we can implement all this `getenv()` stuff into the bookdown.json file as theme settings or something like that. So this should then be the default theme for bookdown?

**Comment by pmjones** [comment]
> Then we can implement all this getenv() stuff into the bookdown.json file as theme settings or something like that

Exactly.

> So this should then be the default theme for bookdown?

Kind of. I'm leaning in favor of something like "bookdown/themes" or similar, where the default project includes that themes package. Does that make sense?

In any case, I'll start up a new repo, and give you control over it; we can change the name etc. as you see fit.

**Comment by codeliner** [comment]
:+1:

**Comment by tobiju** [comment]
> Kind of. I'm leaning in favor of something like "bookdown/themes" or similar, where the default project includes that themes package. Does that make sense?

Yes, this is a good idea. We need also a place to store the assets like css and js. At the moment it is stored at my github page: https://github.com/tobiju/tobiju.github.io

**Comment by pmjones** [comment]
I have created Bookdown.Themes and given you full access. We can talk more as things proceed. Thanks!
